docs/group_faqs/gf_multiple_account.rst

```diff
@@ -0,0 +1,31 @@
+Solace Cloud Multiple Account
+=============================
+
+Overview
+~~~~~~~~
+Users can be added to more than one Solace Cloud Account. Once added, a user can have different access and permission in each environment.
```

Comment:
'permission' should be 'permissions'. Also, access and permissions mean the same thing in this context so maybe reword it to be less redundant?